README.md

```diff
@@ -0,0 +1,118 @@
+# Django-import-export-extensions
+
+| Project   |     | Status |
+|-----------|:----|--------|
+| Versions  |     | [![PyPI - Python Versions](https://img.shields.io/pypi/pyversions/django-import-export-extensions)](https://pypi.org/project/django-import-export-extensions/)<br>[![PyPI - Django Versions](https://img.shields.io/pypi/frameworkversions/django/django-import-export-extensions)](https://pypi.org/project/django-import-export-extensions/) |
+| CI/CD     |     | [![Build status on Github](https://github.com/saritasa-nest/django-import-export-extensions/actions/workflows/checks.yml/badge.svg)](https://github.com/saritasa-nest/django-import-export-extensions/actions/workflows/checks.yml)<br>[![Test coverage](https://coveralls.io/repos/github/saritasa-nest/django-import-export-extensions/badge.svg?branch=main)](https://coveralls.io/github/saritasa-nest/django-import-export-extensions?branch=main)<br>[![Documentation Status](https://readthedocs.org/projects/django-import-export-extensions/badge/?version=latest)](https://django-import-export-extensions.readthedocs.io/en/latest/?badge=latest) |
+| Statistic |     | ![PyPI Downloads](https://static.pepy.tech/badge/django-import-export-extensions/month) |
```

Comment:
In invocations it looks more readable, also let's add extra one(from saritasa invocations)

![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/saritasa-nest/saritasa-invocations/checks.yml)
![PyPI](https://img.shields.io/pypi/v/django-import-export-extensions)
![PyPI - Status](https://img.shields.io/pypi/status/django-import-export-extensions)
![PyPI - Python Version](https://img.shields.io/pypi/pyversions/django-import-export-extensions)
![PyPI - Django Version](https://img.shields.io/pypi/frameworkversions/django/django-import-export-extensions)
![PyPI - License](https://img.shields.io/pypi/l/django-import-export-extensions)
![PyPI - Downloads](https://img.shields.io/pypi/dm/django-import-export-extensions)
[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)

Reply:
Code - yes
Format - I believe that table is more convenient

![image](https://github.com/user-attachments/assets/dfee75d0-45a5-4b09-b637-7ae8cfc51bd2)

Reply:
It's just usually all others do one liner
![image](https://github.com/user-attachments/assets/c291dd43-fd1f-4fd2-8856-8cbaf6d209fd)
![image](https://github.com/user-attachments/assets/62936f20-ca3b-449a-8ae2-406a4d852f1e)
![image](https://github.com/user-attachments/assets/62ba9e3f-dfff-41d3-86c4-8200c25e8fc8)

Reply:
Personally I don't like one liners with lots of items

I suggest making it this way:

![image](https://github.com/user-attachments/assets/04602dab-651a-45f2-aac7-a58c5d038da0)


Simple in code, looks better in UI